src/onnx/onnx.cpp

```diff
@@ -1697,10 +1713,16 @@ struct onnx_parser
             op::lstm{hidden_size, vec_actv_funcs, dirct, clip, input_forget}, std::move(args));
 
         // second output for last lstm output
-        auto last_output = prog.add_instruction(op::rnn_last_output{}, hidden_states);
+        std::vector<instruction_ref> vec_args;
+        vec_args.push_back(hidden_states);
+        if(seq_lens != prog.end())
+        {
+            vec_args.push_back(seq_lens);
```

Comment:
What if `seq_lens` is a constant?

Reply:
Only if seq_lens is a constant and all sequence lengths are the same, we do not need to call these kernels. But if the len is less than the seq_len in the input (of shape [seq_len, batch, input_size], we need to extend the output hidden state to the shape of [seq_len, num_direct, batch, hidden_size] from the shape [len, num_direct, batch, hidden_size] since len < seq_len.